easycukes-scm/src/main/java/com/worldline/easycukes/scm/stepdefs/SCMStepdefs.java

```diff
@@ -45,6 +46,21 @@
 public class SCMStepdefs {
 
     protected static final EasyCukesConfiguration<CommonConfigurationBean> configuration = new EasyCukesConfiguration<>(CommonConfigurationBean.class);
+    
+	/*
+	 *  Used to convert base url into scm base url 
+	 */
+	private static final String SCM_BASE_URL = "/portal";
+	
+	/*
+	 *  Used to log message after commit
+	 */
+	private static final String COMMIT_DONE_MESSAGE = "Done Commit...";

```

Comment:
Actually there were sonar lint issue for this ;-
"Define a constant instead of duplicating this literal". To make this generic and avoid sonar issues Can we configure Error message in future ? 
For now will log messages directly.